src/transaction.js

```diff
@@ -440,7 +440,9 @@ const transaction = {
       throw new MaximumNumberParentsError(`Transaction has ${txData.parents.length} parents and can have at most 2.`);
     }
 
-    txSize += 64 - (32 * (get(txData, 'parents') || 0));
+    // Parents may not be sent in txData
+    const parents = get(txData, 'parents');
```

Comment:
Why don't we do `const parents = txData.parents || []`?

Reply:
Using `get` we prevent any error thrown if `txData` is `null` or `undefined`. We are not adding any new dependencies to the project, so I think it's ok to use the `get`.

Reply:
Ok. But we can still use `|| []` to make sure it is never null, right? Using this we don't have to use the conditional on the line below.

Reply:
Makes sense. Added it